File: neo_crud_2-main/fe/crud-upgrade/src/app/Model/District.ts
export class District {
    id: number;
    name: string;
    provinceId: number;
    village: any;
    constructor(id: number, name: string, provinceId: number, village: any) {
        this.id = id;
        this.name = name;
        this.provinceId = provinceId;
        this.village = village;
    }
}

File: neo_crud_2-main/upgrade-SpringBoot/src/main/java/com/CRUD/demoCRUD/entity/District.java
package com.CRUD.demoCRUD.entity;

import lombok.Data;

import javax.persistence.*;
import java.util.List;

@Entity(name = "tb_district")
@Table(name = "tb_district")
@Data
public class District {
    @Id
    @Column
    public int district_id;
    @Column
    public int province_id;
    @Column
    public String name;

    @OneToMany
    @JoinColumn(name = "district_id",referencedColumnName = "district_id")
    private List<Ward> wardList;


}
